nixos/modules/services/web-servers/meguca.nix

```diff
@@ -102,8 +110,16 @@ in with lib; {
 
       preStart = ''
         # Ensure folder exists or create it and links and permissions are correct
-        mkdir -p ${escapeShellArg cfg.dataDir}
-        ln -sf ${pkgs.meguca}/share/meguca/www ${escapeShellArg cfg.dataDir}
+        mkdir -p ${escapeShellArg cfg.dataDir}/www
+        rm -rf ${escapeShellArg cfg.dataDir}/www/videos
```

Comment:
Not sure how safe it is to try to delete files here

Reply:
Alternatives?

Reply:
I'm not very familiar with the software in question. I think the simplest way would be to make meguca somehow look up files in both cfg.dataDir and in immutable $out/share/meguca.

Reply:
Sounds like a bother, but it's more proper, so I'll make an issue upstream.